src/probnum/diffeq/perturbedsolvers/_perturbation_functions.py

```diff
@@ -0,0 +1,50 @@
+"""perturbation functions to perturb the stepsize."""
+import numpy as np
+
+
+def perturb_uniform(step, order, noise_scale, seed=1):
```

Comment:
1. Instead of `seed=1`, we use `random_state=None` in probnum. This random state is then passed directly to a scipy sampling function, in this case `scipy.stats.uniform.rvs(random_state=random_state)`. Would be great if you could update this :)
2. can you maybe change "order" to "solver_order" or something? there are so many "order"s floating around probnum.diffeq, that it surely helps to be a bit more verbose here.